src/components/ConditionalFilter/filterDefinitions/utils.ts

```diff
@@ -0,0 +1,138 @@
+import { FilterElement } from "../FilterElement";
+import {
+  ConditionValue,
+  isItemOption,
+  isItemOptionArray,
+  isTuple,
+} from "../FilterElement/ConditionValue";
+
+export type ProcessedConditionValue =
+  | string
+  | boolean
+  | string[]
+  | { range: { gte?: string; lte?: string } }
+  | { eq: string }
+  | { oneOf: string[] };
+
+/**
+ * Extracts a boolean value from various input formats.
+ * Handles ItemOption with string values, direct boolean, or stringified values.
+ */
+export function getBooleanValueFromElement(element: FilterElement): boolean {
```

Comment:
use namespace to avoid global scope pollution, eg 

FilterVariablesUtils.getBooleanValueFromElement